wren-ai-service/src/pipelines/retrieval/historical_question.py

```diff
@@ -143,14 +143,17 @@ def __init__(
         embedder_provider: EmbedderProvider,
         store_provider: DocumentStoreProvider,
     ) -> None:
-        self._store = store_provider.get_store(dataset_name="view_questions")
-        self._embedder = embedder_provider.get_text_embedder()
-        self._retriever = store_provider.get_retriever(
-            document_store=self._store,
-        )
-        self._score_filter = ScoreFilter()
-        # todo: add a llm filter to filter out low scoring document
-        self._output_formatter = OutputFormatter()
+        _store = store_provider.get_store(dataset_name="view_questions")
```

Comment:
nitpick: I do believe there is you just forgot removing the underscore. 🤣

Reply:
fixed